typing/typecore.ml

```diff
@@ -6240,25 +6323,25 @@ and type_unpacks ?(in_function : (Location.t * type_expr * bool) option)
                    (mkloc (Longident.Lident unpack.tu_name.txt)
                       unpack.tu_name.loc)))
         in
-        Mtype.lower_nongen (get_level ty) modl.mod_type;
+        (* This call to [lower_nongen] mirrors the corresponding call in typing
+           [Ppx_letmodule]. It's not necessary here, as a first class module
```

Comment:
So should the line be removed? The comment suggests "yes".

Reply:
The thing I was trying to suggest with this comment was: well, if one day module types *can* contain variables, then it might be better to consistently error here (as is accomplished by setting the level to 0) rather than accidentally introduce a soundness bug along the lines of the linked issue.

I also find that it's more natural to comment a line of code rather than the absence of a line of code.

However I did feel conflicted here. Let's talk about this when we talk about https://github.com/ocaml-flambda/ocaml-jst/pull/146#discussion_r1135584250.

Reply:
I removed the line according to our conversation with Leo. I attempted a reword of the comment---I wanted to include an explanation beyond just "this call isn't necessary here", but I fear that my attempt isn't clarifying (and is likely wrong in some ways). Do you have a suggestion?